### ISSUE TITLE
Transfer assignment

### ISSUE THREAD

**Issue by Razpudding** [cmda-be/course-17-18#142]
Awesome! Another interactive tutorial. This one has pretty good feedback and it's easy to follo

Some texts are a bit hard to follow/long/technical

>Where the first part is the protocol and version again, then comes the 
status code (200), and finally the reason phrase (a short description of 
the status code, in this case OK).  After the status line come response 
headers, such as Date (current time on the server) and in the case of 
OPTIONS, Allow (list of allowed methods for the resource).  The response 
head can be followed by data, but in the case of OPTIONS no data is sent 
back from the server.

But I'm guessign this will be explained in a more visual way in the lecture/lab slides so if it's just a recap it might be fine.

Step 9 only works for the exact title mentioned. I guess that's ok but learning process would be better if there were some degrees of freedom. For instance, this way, students might believe nonexistent-movie is a non generic name. I've seen this before with students reusing param names from my code examples.

Step 12: I don't think it's very clear why put would be useful. Maybe a description of a usecase would be useful. Do you use put for instance when you want to save a resource to a location that post would not (be able to)? It also doesn't mention that put OVERWRITES any resource that is already there. I think that's something we should add. BTW I've never actively used curl so this is coming from a beginners perspective. Ive always used the browser for request etc.

Step x: Might be a good idea to add this tip: If you're seeing '<url> malformed' or something similar, it probably means you've made a typo in your request. Are you sure you typed everything correctly and didn't forget any spaces?

**Comment by wooorm** [comment]
- Fixed your nonexistent movie thing
- We’ll talk about put vs post in the lecture
- I feel the malformed part is best to help with in slack/class, it’s a real error people actually see when sending invalid json, so it’s a good way to learn a bit about debugging!

Thanks so much Laurens! 💚

**Comment by Razpudding** [comment]
Sounds good!
I'll try to remember the malformed request thingy in class.
